src/notebook.scss

```diff
@@ -239,6 +239,9 @@ body.notebook {
           height: 0;
           margin: 0 !important;
           padding: 0 !important;
+          position: relative;
```

Comment:
This is a nice pragmatic solution!  I think it might also be good to set it to a larger negative number like `-10000px` and also set `opacity: 0` in the OFF chance that someone creates a extremely long tab - that way it won't overlap.